package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "aepp-showcases",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve --open",
```

Comment:
I'm not exactly happy we're using `vue-cli`, here. It fixes us to an old WebPack version and later migration is more difficult. Also more manual configuration for Babel. We got `@aeternity/aepp-components` to work with WebPack4 in the Beer aepp, FWIW.